### PR TITLE
Create local environment template

### LINKED ISSUE #15: Create Local Development Environment Template

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #15

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-wpaustralia.org
```
Comment (ntwb) [inline CNAME:1]: This file needs to remain until we have a host hosting setup for https://wpaustralia.org

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+The WP Australia site [http://wpaustralia.org/]([http://wpaustralia.org/).
+
+### Installation
+
+We've created a script that automatically sets up a local development server using [Chassis](http://docs.chassis.io/en/latest/).
+
+You can get up and running by doing the following:
+1. Open Terminal.
+2. Change directories into the a folder where you would like to download the project. e.g. `cd /Volumes/Sites`
```
Comment (ntwb) [inline README.md]: `... directories into the folder`, rather than `... directories into the a folder` please

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+3. Run the following command: `curl -L https://git.io/vxWPG | sh`
+4. Wait patiently while the script downloads and sets up the appropriate software.
+5. If all goes well a browser should open [http://wpaustralia.local](http://wpaustralia.local)
+6. You can login to the site by visiting: `http://wpaustralia.local/wp/wp-admin/`(http://wpaustralia.local/wp/wp-admin/). Username: `admin` Password: `password`
+
+### Development Tools
+
+We have bundled a few Chassis Extensions to assist with development. We've included the following extensions:
+* [Xdebug](https://github.com/Chassis/Xdebug) - This extensions automatically sets up the Xdebug php module for you.
+* [Mailhog](https://github.com/Chassis/mailhog) - This extension setup Mailhog which catches all the email WordPress sends. You can see the dashboard for it by visiting. [http://wpaustralia.org:8025/]
```
Comment (ntwb) [inline README.md]: `http://wpaustralia.local:8025/` rather than `http://wpaustralia.org:8025/` please

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
...
+ * [Sequel Pro](https://github.com/Chassis/sequelpro) - This extensions allows you to run `vagrant sequel` from the command line and it will generate a `.spf` file and open the mySQL database in [Sequel Pro](https://www.sequelpro.com/).
+* [Database Backup](https://github.com/Chassis/db-backup/) - This extension will prompt you to backup your database if you run `vagrant destroy` from the command line.
+* [Fish](https://github.com/Chassis/Fish/) - This extension replaces Bash shell with [Fish](https://fishshell.com/) shell.
+
+## Useful commands
+
+* `vagrant up` - Run this in the directory you installed this project in to boot up your development environment. e.g. `cd /Volumes/Sites/wpaustralia; vagrant up` then browse to [http://wpaustralia.local](http://wpaustralia.local)
```
Comment (ntwb) [inline README.md]: "Run this from the `wpaustralia` directory where you downloaded and installed this project"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,43 @@
...
+* `vagrant halt` - Run this when you have finished development to shutdown the Vagrant Virtual Machine. e.g. `cd /Volumes/Sites/wpaustralia; vagrant halt`
+* `vagrant destroy` - Run this if you wish to destory the Vagrant Virtual Machine.
+* `vagrant provison` - Run this if you have added an additional Chassis extension or if you are [disabling](http://docs.chassis.io/en/latest/config/#extensions) one of the Chassis extensions.
+
+### Development Process
+
+If you're new to give them here are a few steps to help you get started:
```
Comment (ntwb) [inline README.md]: ???

**Review by ntwb — CHANGES_REQUESTED** [review]
As per my comment on the `CNAME` file deletion, we can't delete any of these files until we have DNS pointed at out new host (which at the time of writing we do not currently have)

**Comment by BronsonQuick** [comment]
@ntwb Thanks for the review. Sorry for lots of horrible English as I was rushing and didn't proofread anything 🤣

I've added a new branch called `development` and changed this PR to merge into that. That way we can keep the CNAME stuff for Github in `master` and work on adding themes and plugins in development then when we're ready to switch to a WordPress install on the site we can merge development into master and alter the workflow.

**Comment by ntwb** [comment]
Thanks a bunch for the PR and follow up 💐

**Review by ntwb — APPROVED** [review]
LGTM (Looks Good To Me) 👍

**Pull request merged**